Formula/lima.rb

```diff
@@ -0,0 +1,30 @@
+class Lima < Formula
+  desc "Linux-on-Mac - macOS subsystem for Linux - containerd for Mac"
+  homepage "https://github.com/AkihiroSuda/lima"
+  url "https://github.com/AkihiroSuda/lima.git",
+    tag:      "v0.2.1",
+    revision: "4ccd40976296a8911ad84fddeff82bb9c2d754bd"
+  license "Apache-2.0"
+  head "https://github.com/AkihiroSuda/lima"
+
+  depends_on "go" => :build
+
+  depends_on "qemu"
+
+  def install
+    arch = Hardware::CPU.arm? ? "arm64" : "amd64"
+
+    system "GOOS=darwin GOARCH=#{arch} make clean binaries"
+    bin.install Dir["_output/bin/*"]
+    share.install Dir["_output/share/*"]
+
+    # Install bash completion
+    output = Utils.safe_popen_read("#{bin}/limactl", "completion", "bash")
+    (bash_completion/"limactl").write output
+  end
+
+  test do
+    s_output = shell_output("#{bin}/limactl 2>&1")
+    assert_match "NAME:", s_output
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Thx for the reply, I read the documentation before I created the commit.
I thought about the topic again and I changed the test to a mother execution method of lima.

Using a file as a Input was not the best way for that tool so hope the new test looks better for you.

Reply:
`limactl validate examples/*.yaml` may make more sense. (Available in Lima v0.3.0)

https://github.com/AkihiroSuda/lima/pull/48